docs/developer-docs/integrations/sns/managing/sns-asset-canister.md

```diff
@@ -24,11 +24,16 @@ An asset canister must first be deployed before control of it can be handed over
 The general overview of deploying an asset canister during an SNS launch is as follows:
 - First, the asset canister must be created with or upgraded to a Wasm file from [dfx 0.14.1+](https://github.com/dfinity/sdk/blob/release-0.14.1/src/distributed/assetstorage.wasm.gz).
 - Then, the dapp should hand control of the asset canister over to the SNS by setting the following permissions:
-    - The SNS governance canister is given `Commit` permissions. This is done by the previous developer using the `grant_permissions` command (see the granting permissions section below), otherwise the SNS must grant this permission once it is a controller.
+    - The SNS governance canister should be given all permissions before handing the canister over. The SNS governance canister can be manually added to all necessary permissions, including `ManagePermissions`, or it can call the `take_ownership` method to grant the necessary permissions.
```

Comment:
`take_ownership` only grants `Commit` permissions to the caller, not `ManagePermissions`.